src/components/searchBar/SearchBar.tsx

```diff
@@ -7,9 +7,9 @@ export interface SearchBarProps extends InputGroupProps {}
 
 const SearchBar = ({ children, ...rest }: SearchBarProps) => {
   return (
-    <div className="d-flex">
-      <InputGroup {...rest}>{children}</InputGroup>
-    </div>
+    <InputGroup className="d-flex flex-nowrap search-input-group" {...rest}>
```

Comment:
I think this being an input group results in the input having no right border

![image](https://github.com/the-curve-consulting/texmo-react-components/assets/73468976/74e22f3f-7ba8-4362-9a74-69c942f08e09)